data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,353 @@
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You are not sure if they are being serious or not, but their friends seem to be in for it. What seems to be their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" Another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, this is not a friendly competition between allies. As far as I am concerned you are an enemy. You should see it more as a fight for you to truly earn this ship, and more, but make no mistakes: it has only one rule, and that is that nobody else than us may interfere."`
+			# this is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles
+			# (but in reality with disabling a lot of ships it goes faster than that)
+			`	"Although sometimes the rules can bend a little. But you certainly did not earn that privilege."`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	The engineer comes back, looking a bit impatient, "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships to get the initiative. The engineer utters a word to you once they are far enough away, "I want this done as quickly as possible, do not try my patience. You are lucky I offered this deal in the first place, make sure I do not regret it. Now, quick, get to your ship!" You see the contestants have already begon the launching procedures, and launch as soon as you can.`
+				launch
+	on decline
+		"reputation: Hai (Unfettered)" >?= -1000
+	on fail
+		"reputation: Hai (Unfettered)" >?= -1000
+		dialog `The Unfettered hail you, "You failed the mission, and as such, are no longer authorized to land on our worlds, and you will not enjoy unmerited respect any longer!"`
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+		to despawn
+			"total ships" != 1
+		personality unconstrained launching plunders staying target heroic disables
+		ship "Lightning Bug (Ionic)" "Erazer"
+		ship "Lightning Bug" "Thunder"
+		ship "Lightning Bug (Unfettered Shipyards)" "Lighting"
+		ship "Lightning Bug (Pulse)" "Struck"
+		dialog "You receive a message from the remaining Unfettered ships: they surrender. They will not wait on the planet forever, however, so you need to land on this day to claim the rewards."
+		on disable
+			set "unfettered: beaten the bugs"
```

Comment:
Why is this needed? You already have to disable them to complete (`npc disable`)..

Reply:
to make sure you dont win because they dont spawn

Reply:
Thanks!

Reply:
I just checked I cant just have `set X` I need it in the on disable